fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractPersistenceTest.java

```diff
@@ -113,15 +113,13 @@ public void setUp() throws Exception {
         // Note: this assumes that the concrete test classes will be in a project that is peer to the fhir-persistence module
         // TODO: it would be better for our unit tests if we could load config files from the classpath
         FHIRConfiguration.setConfigHome("../fhir-persistence/target/test-classes");
```

Comment:
I think we'd still want to set the configHome on each unit tests that need to read fhir-server-config from disk, so I'm not sure it would buy us much.  Maybe it would be better to clear it at the end of each test that sets it, to ensure each test is actually setting it?

Anyway, I agree that a pluggable FHIRConfiguration strategy is probably the right answer here...we have https://github.com/IBM/FHIR/issues/1318 for that